### PR TITLE
Feature/us 014 at sce api container

### PR DESCRIPTION
**Pull request by ATdllerenas**
Here is the at-sce-api Container, The second commit was the modification of the packages  aren't seen inside the container, the third commit was the change from UserRepository Class to UserRepository Interface and the same with UserServices. I didn't know the summary was the update of each change.

### CODE REVIEW

**Review by luisgalindoluna — APPROVED** [review]
Looks good

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks good

File: at-sce-api/src/test/java/com/example/atsceapi/AtSceApiApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.example.atsceapi;
```
Comment (ivanflores15) [inline at-sce-api/src/test/java/com/example/atsceapi/AtSceApiApplicationTests.java]: Why package name is com.example?

**Review by ivanflores15 — COMMENTED** [review]
@luisgalindoluna I think all the folder under .mvn/** should not be included, should be?

**Comment by luisgalindoluna** [comment]
> @luisgalindoluna I think all the folder under .mvn/** should not be included, should be?

@ivanflores15 It shouldn't? As far as I could tell those files were created automatically because of spring init, I don't think they should cause any problems.

**Review by ivanflores15 — COMMENTED** [review]
The convention for package naming is use lowercase, that means name of package should be com.example.atsceapi.**service** instead of com.example.atsceapi.**Service**

**Pull request closed without merging**